libs/nestjs-admin/src/views/index.njk

```diff
@@ -1,23 +1,31 @@
 {% set useJquery = false %}
 {% extends 'base.njk'%}
 
+{% block stylesheets %}
+<link rel="stylesheet" href="/static/css/index.css">
+{% endblock %}
+
 {% block body %}
 
-<ul>
-  {% for section in sections %}
-    <li>
-      <h2>{{ section.name }}</h2>
-      <ul>
-        {% for entityMD in section.getEntitiesMetadata() %}
-          <li>
+{% for section in sections %}
+  <div class="mt-4">
```

Comment:
I'd rather add explicit margin on the `.section-table`, easier to extend than hardcoding the margins in the HTML